src/tests/unit/tests/DetailsView/components/start-over-dropdown.test.tsx

```diff
@@ -82,78 +143,25 @@ describe('StartOverDropdownTest', () => {
                 defaultProps.rightPanelOptions = rightPanelOptions;
                 defaultProps.switcherStartOverPreferences = switcherPreferences;
 
-                render(<StartOverDropdown {...defaultProps} />);
-                await act(() =>
-                    getMockComponentClassPropsForCall(InsightsCommandButton).onClick({
-                        currentTarget: 'test target',
-                    }),
-                );
-
+                const renderResult = render(<StartOverDropdown {...defaultProps} />);
+                fireEvent.click(renderResult.getByRole('button'));
                 const isStartOverOptionRendered = getMockComponentClassPropsForCall(
-                    ContextualMenu,
-                ).items.some(item => item.key === optionKey);
+                    MenuList,
+                ).children.some(item => item.key === optionKey);
                 expect(isStartOverOptionRendered).toEqual(shouldFindOption);
             });
         });
     });
 
-    it('render with dropdown on left', async () => {
```

Comment:
Where did this test go?

Reply:
Hi @madalynrose yeah its available in the file, just its moved up.
![image](https://github.com/user-attachments/assets/74527e01-1b6a-4878-a27b-0fc8e9dc8086)